pkg/sql/conn_executor_test.go

```diff
@@ -91,32 +91,20 @@ INSERT INTO sensitive(super, sensible) VALUES('that', 'nobody', 'must', 'see')
 		t.Errorf("wanted: %s\ngot: %s", expMessage, actMessage)
 	}
 
-	const expSafeRedactedMessage = `some error
-(1) while executing: INSERT INTO _(_, _) VALUES ('_', '_', __more2__)
-Wraps: (2) attached stack trace
-  -- stack trace:
-  | github.com/cockroachdb/cockroach/pkg/sql_test.TestAnonymizeStatementsForReporting
-  | 	...conn_executor_test.go:NN
-  | testing.tRunner
-  | 	...testing.go:NN
-  | runtime.goexit
-  | 	...asm_scrubbed.s:NN
-Wraps: (3) some error
-Error types: (1) *safedetails.withSafeDetails (2) *withstack.withStack (3) *errutil.leafError`
+	const expSafeRedactedMsgPrefix = `some error
+(1) while executing: INSERT INTO _(_, _) VALUES ('_', '_', __more2__)`
 
 	// Edit non-determinstic stack trace filenames from the message.
-	actSafeRedactedMessage := strings.ReplaceAll(strings.ReplaceAll(fileref.ReplaceAllString(
-		redact.Sprintf("%+v", safeErr).Redact().StripMarkers(), "...$2:NN"),
-		"asm_arm64", "asm_scrubbed"),
-		"asm_amd64", "asm_scrubbed")
+	actSafeRedactedMessage := fileref.ReplaceAllString(
```

Comment:
is this still necessary now that we only look at the prefix

Reply:
The regexp replacement is unnecessary indeed. 
The chain `.Redact().StripMarkers()` also seems unnecessary because sql.WithAnonymizedStatement already seems to have safely redacted and stripped.